examples/coap-server.c

```diff
@@ -375,9 +374,11 @@ get_context(const char *node, const char *port) {
       memcpy(&addr.addr, rp->ai_addr, rp->ai_addrlen);
       addrs = addr;
       if (addr.addr.sa.sa_family == AF_INET) {
-        addrs.addr.sin.sin_port = htons(ntohs(addr.addr.sin.sin_port) + 1);
+        u_int16_t temp = ntohs(addr.addr.sin.sin_port) + 1;
```

Comment:
Please use the standard type `uint16_t` here.

Reply:
Fixed in update